test/cases/filters.coffeescript.jade

```diff
@@ -1,6 +1,6 @@
 script(type='text/javascript')
-  :coffeescript
+  :coffee-script
     regexp = /\n/
-  :coffeescript(minify=true)
+  :coffee-script(minify=true)
```

Comment:
Added some additional `inputFormats` to the coffeescript transform:
https://github.com/jstransformers/jstransformer-coffee-script/commit/d954505f2e3e99b53b99cb3479829b9ede04e95a